Samples/Docker-Simple/Client/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using GrainInterfaces;
+using Orleans;
+using Orleans.Hosting;
+using Orleans.Runtime.Configuration;
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Client
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var connectionString = File.ReadAllText("connection-string.txt");
```

Comment:
I would recommend leveraging the Microsoft.Extensions.Configuration system to read either from an INI or json file. We are going to be endorsing this more and more as we get rid of the legacy XML configuration.

Reply:
Yes I know, but this is a simple sample, I find it easier to just drop the content of your connection string in a file than replacing a string in a previously existing json file.